Tasks/XamariniOS/xamarinios.ts

```diff
@@ -147,9 +148,14 @@ nugetRunner.exec()
                                             .then(function (code) {
                                                 tl.exit(code);
                                             })
+                                    } else {
+                                        tl.exit(0);
```

Comment:
Well, this task was already using exit. But not if the else cases were hit. This causes the result not to get set and was evident in the test. This was the simplest fix.